Translate "Create a New React App" page (#35)
reactjs/fr.reactjs.org: content/docs/create-a-new-react-app.md

Apprenez Next.js grâce à [son guide officiel](https://nextjs.org/learn/).

### Gatsby {#gatsby}

[Gatsby](https://www.gatsbyjs.org/) est la meilleure option pour créer des **sites web statiques** avec React. Il vous permet d'utiliser des composants React, mais génère du HTML et du CSS pré-rendus afin de garantir le temps de chargement le plus rapide.

Apprenez Gatsby grâce à [son guide officiel](https://www.gatsbyjs.org/docs/) et à une [collection de kits de démarrage](https://www.gatsbyjs.org/docs/gatsby-starters/).

### Chaînes d'outils plus flexibles {#more-flexible-toolchains}

Les chaînes d'outils suivantes offrent plus de flexibilité et de choix. Nous les recommandons pour les utilisateurs expérimentés :

- **[Neutrino](https://neutrinojs.org/)** combine la puissance de [webpack](https://webpack.js.org/) avec la simplicité des préréglages. Il inclut un préréglage pour les [applications React](https://neutrinojs.org/packages/react/) et les [composants React](https://neutrinojs.org/packages/react-components/).

> - **[nwb](https://github.com/insin/nwb)** est particulièrement utile pour [publier des composants React pour npm](https://github.com/insin/nwb/blob/master/docs/guides/ReactComponents.md#developing-react-components-and-libraries-with-nwb). Il [peut également être utilisé](https://github.com/insin/nwb/blob/master/docs/guides/ReactApps.md#developing-react-apps-with-nwb) pour créer des applications React.

Reviewer: ```suggestion
- **[nwb](https://github.com/insin/nwb)** est particulièrement utile pour [publier des composants React sur npm](https://github.com/insin/nwb/blob/master/docs/guides/ReactComponents.md#developing-react-components-and-libraries-with-nwb). Il [peut également être utilisé](https://github.com/insin/nwb/blob/master/docs/guides/ReactApps.md#developing-react-apps-with-nwb) pour créer des applications React.
```

La VO est mal fichue, les composants ne sont pas « pour » npm, mais pour *publication* « sur » npm.

> - **[Parcel](https://parceljs.org/)** est un empaqueteur d'applications web rapide et sans configuration qui [fonctionne avec React](https://parceljs.org/recipes.html#react).

Reviewer: ```suggestion
- **[Parcel](https://parceljs.org/)** est un *bundler* d'applications web rapide et sans configuration qui [fonctionne avec React](https://parceljs.org/recipes.html#react).
```

*Trop mignon* 😄 , « empaqueteur », mais vraiment trop peu répandu à ce stade en VF, qui reste sur “bundler” dans l’usage courant…

- **[Razzle](https://github.com/jaredpalmer/razzle)** est un framework de rendu côté serveur qui ne requiert aucune configuration, mais offre plus de flexibilité que Next.js.

> ## Créer une chaîne d'outils à partir de rien {#creating-a-toolchain-from-scratch}

Reviewer: ```suggestion
## Créer une chaîne d'outils à partir de zéro {#creating-a-toolchain-from-scratch}
```

On traduit généralement “from scratch” par « à partir de zéro », qui ne sous-entend pas le dénuement qu’on perçoit dans « de rien ».

> Une chaîne d'outils de construction en JavaScript consiste généralement en :

Reviewer: ```suggestion
Une chaîne d'outils de construction en JavaScript comprend généralement :
```

On a une découpe plus claire comprend / consiste en français que le bordel consist / comprise en anglais, profitons-en.

* Un **gestionnaire de paquets**, tel que [Yarn](https://yarnpkg.com/) ou [npm](https://www.npmjs.com/). Il vous permet de tirer parti d'un vaste écosystème de paquets tiers, et de les installer ou les mettre à jour facilement.

> * Un **empaqueteur**, tel que [webpack](https://webpack.js.org/) ou [Parcel](https://parceljs.org/). Il vous permet d'écrire du code modulaire et de le regrouper en petits paquets permettant d'optimiser le temps de chargement.

Reviewer: ```suggestion
* Un **_bundler_**, tel que [webpack](https://webpack.js.org/) ou [Parcel](https://parceljs.org/). Il vous permet d'écrire du code modulaire et de le regrouper en petits paquets permettant d'optimiser le temps de chargement.
```

Encore une fois trop mignon, mais vraiment trop exotique à ce stade.

Author: J'ai hésité à garder _bundler_, mais je me suis laissé convaincre par MDN (qui ne l'a certes utilisé [qu'une seule fois](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Instructions/import#Syntaxe))

> * Un **compilateur** tel que [Babel](http://babeljs.io/). Il vous permet d'écrire du JavaScript moderne qui continuera à fonctionner dans les navigateurs les plus anciens.

Reviewer: ```suggestion
* Un **compilateur** tel que [Babel](http://babeljs.io/). Il vous permet d'écrire du JavaScript moderne qui fonctionnera quand même dans les navigateurs les plus anciens.
```

Bizarre la formulation de base, qui dit « continuer à » (orienté avenir) puis une cible passée…

> Si vous préférez configurer votre propre chaîne d'outils JavaScript à partir de rien, [consultez ce guide](https://blog.usejournal.com/creating-a-react-app-from-scratch-f3c693b84658) qui re-crée certaines des fonctionnalités de Create React App.

Reviewer: ```suggestion
Si vous préférez configurer votre propre chaîne d'outils JavaScript à partir de zéro, [jetez un œil à ce guide](https://blog.usejournal.com/creating-a-react-app-from-scratch-f3c693b84658) qui re-crée certaines des fonctionnalités de Create React App.
```

On traduit le plus souvent “check out” par « jeter un œil à », surtout si c'est une invitation à consulter une doc connexe à titre optionnel.

> N'oubliez pas de vous assurer que votre chaîne d'outils personnalisée [est correctement configurée pour la production](/docs/optimizing-performance.html#use-the-production-build).

Reviewer: ```suggestion
Pensez à vous assurer que votre outillage personnalisé [est correctement configuré pour la production](/docs/optimizing-performance.html#use-the-production-build).
```

Souvent plus efficace en invite positive (« pensez ») que négative (« n’oubliez pas »), surtout avant une locution un peu alambiquée comme ici.
